src/store/configureStore.js

```diff
@@ -0,0 +1,23 @@
+import {createStore, combineReducers, applyMiddleware, compose} from 'redux';
+import thunk from 'redux-thunk';
+
+import filtersReducer from '../reducers/filters';
+import movieReducer from '../reducers/movie';
+import moviesReducer from '../reducers/movies';
+import suggestionsReducer from '../reducers/suggestions';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+export default () => {
+  const store = createStore(
+    combineReducers({
+      movies: moviesReducer,
```

Comment:
I believe movies and movie could be just one reducer to make easier your life dealing with your state